sdcm/utils/aws_utils.py

```diff
@@ -311,3 +311,9 @@ def ec2_ami_get_root_device_name(image_id, region):
     except (TypeError, ClientError) as exc:
         raise AssertionError(f"Image '{image_id}' details not found in '{region}'") from exc
     return None
+
+
+def get_arch_from_instance_type(instance_type: str) -> AwsArchType:
+    if instance_type.startswith('a1.'):
```

Comment:
that is not the only type of ARM instances..
in my test i'm using `im4gn.2xlarge` and also i know about `is4gen.4xlarge`

Reply:
there might be others as well

Reply:
thanks. fixed